src/style.css

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search__input {
+  width: 100%;
+  outline: none;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+}
+
+.search__input::placeholder {
+  font-weight: 300;
+}
+
+.search__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search__input:focus {
+  background-color: transparent;
+  border-radius: 3px;
+}
+
+.bigSearch {
+  margin: 20px 0;
+}
+
+.search__form {
+  background: linear-gradient(180deg, #fff, #f6f6f7);
+}
+
+.bigSearch__input {
```

Comment:
you can add this style to search-input class, and change the style for small input (add new style for search-input--small)

search-input and search-input--small class better by BEM